src/utils/remove.test.ts

```diff
@@ -0,0 +1,34 @@
+import remove from './remove';
+
+describe('remove()', () => {
+    let text: string;
+    let pattern: string;
+
+    beforeEach(() => {
+        text = 'this is my test string';
+        pattern = 'test';
+    });
+
+    test('should remove pattern from text', () => {
+        const check = remove(text, pattern);
+        expect(check).toBe('this is my  string');
+    });
+
```

Comment:
I would think that it shouldn't create an extra space. I know why it does it, but don't you think it's an unwanted side effect?

Reply:
I actually left it like that, but I see your point. I've changed it now, double white space removed.